ansible_bender/builders/buildah_builder.py

```diff
@@ -65,6 +65,23 @@ def podman_run_cmd(container_image, cmd, log_stderr=True, return_output=False):
     return run_cmd(["podman", "run", "--rm", container_image] + cmd,
                    return_output=return_output, log_stderr=log_stderr)
 
+def buildah_run_cmd(container_image, container_name, cmd, log_stderr=True, return_output=False):
+    """
+    run provided command in selected container image using buildah; raise exc when command fails
+
+    :param container_image: str
+    :param container_name: str
+    :param cmd: list of str
+    :param log_stderr: bool, log errors to stdout as ERROR level
+    :param return_output: bool, if True, return output of the command
+    :yield: stdout output
+    """
+    create_buildah_container(container_image, container_name, None, None, False)
+    yield run_cmd(["buildah", "run", container_name] + cmd,
```

Comment:
please drop the yield - if you got inspired from tests, that's how pytest handles fixtures

for normal python code, this should be something like:
```python
try:
  buildah run
finally:
  buildah rm
```